src/lib/functions/server.test.js

```diff
@@ -17,6 +18,9 @@ const functionsPath = `functions`
 /** @type { express.Express} */
 let app
 
+// mock the file-system to not write on the real one with `mkdir` inside the registry.js
+mock({ node_modules: mock.load(join(__dirname, '../../../node_modules')) })
```

Comment:
tests were failing before as we were doing an mkdir un-mocked on the real fs which is not possible to write inside `/my-functions/functions` on CI therefore I mocked the entire filesystem as the test does not need one.